GOALS.md

```diff
@@ -3,13 +3,13 @@
 
 The ultimate goal is to make the development of apps adhering to [WAI-ARIA](http://www.w3.org/TR/wai-aria/) a breeze. The ultimatest™ goal is to make this library obsolete by lobbying the W3C and browser vendors into supporting the basic tools provided by ally.js. (One may dream…)
 
-Applications have to deal with quite a number of different issues to get to a state we may call "accessible". There's the issue of *maintaining the focus*, a subset of providing *keyboard navigation*. Then there's [WAI-ARIA](http://www.w3.org/TR/wai-aria/) to provide help the browser provide the operating system's Accessibility Infrastructure with the necessary details.
+Applications have to deal with quite a number of different issues to get to a state we may call "accessible." There's the issue of *maintaining the focus*, a subset of providing *keyboard navigation*. Then there's [WAI-ARIA](http://www.w3.org/TR/wai-aria/), which helps the browser provide the operating system's Accessibility Infrastructure with the necessary details.
 
-Documents like [WAI-ARIA Authoring Practices](http://www.w3.org/WAI/PF/aria-practices/) try to explain what needs to be done, but even though the documents have been "dumbed down", they're still too convoluted to allow quickly understanding things. With `ally.js` I want to take that english language document and turn it into generic, reusable JavaScript.
+Documents like [WAI-ARIA Authoring Practices](http://www.w3.org/WAI/PF/aria-practices/) try to explain what needs to be done, but, even though the documents have been "dumbed down", they're still too convoluted to allow quickly understanding things. With `ally.js` I want to take that English language document and turn it into generic, reusable JavaScript.
 
-It does not make sense to implement the same things over and over again, which is why `ally.js` is designed in a way that while allowing simple and drect integration (like jQuery), exposes every function individually. The modularity is paramount for preventing application and library authors from copy-pasting code, or worse, rolling their own flawed implementation. While ally.js is not perfect (it's software, it's written by me, …), I want to make this *the place we focus our efforts* - much like has happened for the DOM by way of jQuery.
+It does not make sense to implement the same things over and over again, which is why `ally.js` is designed in a way that, while allowing simple and direct integration (like jQuery), exposes every function individually. The modularity is paramount for preventing application and library authors from copy-pasting code or, worse, rolling their own flawed implementations. While ally.js is not perfect (it's software, it's written by me, …), I want to make this *the place we focus our efforts* - much like has happened for the DOM by way of jQuery.
```

Comment:
the diff doesn't make it clear, but this is what I changed:

`drect` ➔ `direct`
`that while allowing` ➔ `that, while allowing`
`code, or worse, rolling` ➔ `code or, worse, rolling`
`flawed implementation.` ➔ `flawed implementations.`